Dockerfile

```
@@ -1,7 +1,6 @@
 FROM archlinux/base
 
-RUN pacman -Syu --noconfirm
-RUN pacman -S --noconfirm --needed --overwrite '*' \
+RUN pacman -Syu --noconfirm --needed --overwrite '*' \
```

```suggestion
RUN pacman -Sy --noconfirm --needed --overwrite '*' \
```

If you don't upgrade all packages, but just what you're about to install (you're basing off of archinux/base, so you can't be too far behind) - things might work better. I've just tried this and it works. I'm assuming there's an upgrade that breaks `/var/lib/pacman` and `/etc/makepkg.conf` - I'm not sure what yet.